README.md

```diff
@@ -1,9 +1,10 @@
 # Limber Pipeline Application
 
-[![Build Status](https://travis-ci.org/sanger/limber.svg?branch=next_release)](https://travis-ci.org/sanger/limber)
+![Linting](https://github.com/JamesGlover/limber/workflows/Linting/badge.svg)
+![Ruby RSpec Tests](https://github.com/JamesGlover/limber/workflows/Ruby%20RSpec%20Tests/badge.svg)
+![Javascript testing](https://github.com/JamesGlover/limber/workflows/Javascript%20testing/badge.svg)
```

Comment:
similar to on other PR - referring to your repo - should probably refer to Sanger

Reply:
Ooh, good catch.